pkg/controllers/cspc-controller/sync.go

```diff
@@ -438,3 +458,17 @@ func (pc *PoolConfig) patchPoolDeploymentSpec(cspi *cstor.CStorPoolInstance) err
 
 	return nil
 }
+
+func (c *Controller) ShouldReconcile(cspc cstor.CStorPoolCluster) (bool, string) {
```

Comment:
Why return type string?Can we have a return type error?

Reply:
As discussed -- we want to have the reason of truthy value and not error occurs here.